wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -716,15 +717,43 @@ def timesince_last_update(
         else:
             time_str = last_update.strftime("%H:%M")
 
-        time_prefix = f"{time_prefix} " if time_prefix else time_prefix
-        by_user = f" by {user_display_name}" if user_display_name else user_display_name
-
-        return f"{time_prefix}{time_str}{by_user}"
-
+        if show_time_prefix:
+            if user_display_name:
+                return _("at %(time)s by %(user_display_name)s") % {
```

Comment:
Not critical for this PR now, but do you think this area of code would be a good candidate for translators comments? (example below)

I do not do translations myself so maybe these are clear enough as is but thinking it might be a good place to try this out and get feedback from translators.

https://docs.djangoproject.com/en/4.1/topics/i18n/translation/#comments-for-translators

```suggestion
                # Translators: used when time is today
                return _("at %(time)s by %(user_display_name)s") % {
```

Reply:
Sry, I'm too late: those comments could be helpful, in that specific case it probably doesn't really change anything, because the string, using "time" as a placeholder, already speaks for itself.
So merging this PR without that change is ok for me, thanks btw! :smile:

Reply:
Looking at it again, what may help as a comment is, that the versions with the prefix are normally appended to other strings. Personally, if I see such a string, I normally look into the code and check its usages to get an overview.

Reply:
Yeah - it may be overkill to use the translator comments. We don't use them anywhere else... Yet. 

Just thought it might be a good use case to try them out and get feedback from translators. 

But again. Not critical and this has been merged and is fine without them. 

Thanks for digging into all this again.